documentation/notes/joss_paper/joss.bib

```diff
@@ -161,14 +161,26 @@ @misc{sparse
 }
 
 @article{vCV2011,
-  author={S. {van der Walt} and S. C. {Colbert} and G. {Varoquaux}},
-  journal={Computing in Science Engineering},
-  title={The NumPy Array: A Structure for Efficient Numerical Computation},
-  year={2011},
-  volume={13},
-  number={2},
-  pages={22-30},
-  doi={10.1109/MCSE.2011.37}
+ title         = {Array programming with {NumPy}},
```

Comment:
Hopefully this one is self-explanatory, but there is a much newer reference from a few months ago (after the start of the review) that [NumPy now suggest to use](https://numpy.org/citing-numpy/) as a citation, so I have lifted this suggestion from that. It would be best to use this up-to-date citation.

Reply:
Yes thank you.